GUI/SearchTaskWindow.xaml.cs

```diff
@@ -353,6 +352,19 @@ private void SaveButton_Click(object sender, RoutedEventArgs e)
             }
 
             Protease protease = (Protease)proteaseComboBox.SelectedItem;
+            if (classicSemiSpecific)
```

Comment:
I assume you're including options for "traditional" semi specific and your semi specific search algorithm. This seems pretty confusing. If a user wants to do a semi-specific search will it be clear in the GUI what the difference is and what they're supposed to do?

I'd rather we just give one option instead of providing one recommended one and one subpar one with the same names

Reply:
I completely agree. I need both, for now, to compare between them for the paper.